wazo_phoned/plugins/endpoint/services.py

```diff
@@ -38,6 +38,16 @@ def unhold(self, endpoint_name):
             logger.debug('No plugin found for endpoint %s', endpoint_name)
             raise NowhereToRouteEndpoint(endpoint_name)
 
+    def answer(self, endpoint_name):
+        plugin = self.route_to_plugin(endpoint_name)
+        if plugin:
```

Comment:
looking at answer, unhold and hold makes me wonder if route_to_plugin should raise of not plugin. I did not check if there are other usages of this function.